editor/src/clj/internal/node.clj

```diff
@@ -1029,7 +1029,7 @@
                                                                (update :default f)
 
                                                                (some? (-> propdef :dynamics))
-                                                               (update :dynamics (partial util/map-vals f)))]))
+                                                               (update :dynamics (partial coll/map-vals f)))]))
```

Comment:
I've moved `util/map-vals` into `util.coll`, but kept `util/map-vals` as a macro so this PR wouldn't baloon in size. However, we used it as a function in this one file.